google-cloud-bigquerystorage/pom.xml

```diff
@@ -108,6 +108,12 @@
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-lang3</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.google.zetasql</groupId>
```

Comment:
is this a credibly sourced artifact? do we have a more well-known alternative to this?